docs/getting-started.rst

```diff
@@ -161,74 +158,45 @@ Columns returned by the simulator
-+------------------------+-----------------------------------------------------+-------+
-|kiz                     |Monthly additional child benefit, household sum      | float |
-+------------------------+-----------------------------------------------------+-------+
-|wohngeld                |Monthly housing benefit, household sum               | float |
-+------------------------+-----------------------------------------------------+-------+
-|m_alg2                  |Monthly social assistance, household sum             | float |
-+------------------------+-----------------------------------------------------+-------+
-|dpi_ind                 |Monthly disposable income, individual                | float |
-+------------------------+-----------------------------------------------------+-------+
-|dpi                     |Monthly disposable income, household sum             | float |
-+------------------------+-----------------------------------------------------+-------+
-|gross                   |Monthly market income                                | float |
-+------------------------+-----------------------------------------------------+-------+
++-------------------------+----------------------------------------------------+-------+
+|    Variable             | Explanation                                        | type  |
++=========================+====================================================+=======+
+| rentenv_beit_m          | Monthly amount employee old-age pensions contrib.  | float |
++-------------------------+----------------------------------------------------+-------+
+| arbeitsl_v_beit_m       | Monthly amount employee unempl. insurance contrib. | float |
++-------------------------+----------------------------------------------------+-------+
+| ges_krankv_beit_m       | Monthly amount employee health insurance contrib.  | float |
++-------------------------+----------------------------------------------------+-------+
+| pflegev_beit_m          | Monthly amount of long term care insurance         | float |
++-------------------------+----------------------------------------------------+-------+
+| arbeitsl_geld_m         | Monthly amount of unemployment assistance          | float |
++-------------------------+----------------------------------------------------+-------+
+| rente_anspr_m           | Monthly amount of old-age pensions                 | float |
++-------------------------+----------------------------------------------------+-------+
+| abgelt_st               | Monthly capital income tax due, individual         | float |
++-------------------------+----------------------------------------------------+-------+
+| soli_st                 | Monthly solidarity surcharge due, individual       | float |
++-------------------------+----------------------------------------------------+-------+
+| kindergeld              | Monthly child Benefit, individual                  | float |
```

Comment:
Make this `kindergeld_m` (I would think of the general rule for now: Assume annually unless specified otherwise).

Reply:
Has been changed to `kindergeld_m`.